core-dev/packages/coqide/coqide.8.10.dev/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, and contributors."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "git+https://github.com/coq/coq.git"
+license: "LGPL-2.1"
+
+depends: [
+  "coq" {= version}
+  "lablgtk3-sourceview3"
+]
+build: [
+  [
+    "./configure"
+    "-configdir" "%{lib}%/coq/config"
+    "-prefix" prefix
+    "-mandir" man
+    "-docdir" doc
+    "-libdir" "%{lib}%/coq"
+    "-datadir" "%{share}%/coq"
+  ]
+  [make "-j%{jobs}%" "coqide-files"]
+  [make "-j%{jobs}%" "coqide-opt"]
+]
+install: [
+  make
+  "install-ide-bin"
+  "install-ide-files"
+  "install-ide-info"
+  "install-ide-devfiles"
+]
+remove: ["rm" "-rf" "%{lib}%/coq/ide" "%{doc}%/FAQ-CoqIde"]
+synopsis: "IDE of the Coq formal proof management system"
+flags: light-uninstall
```

Comment:
Please remove `flags: light-uninstall` and the `remove` line. They are no longer needed since OPAM 2 tracks installed files.

Reply:
Thanks! I also did this for the `coq` packages, while I was at it.